back/Gemfile.lock

```diff
@@ -310,11 +310,6 @@ PATH
       rails (~> 6.1)
       ros-apartment (>= 2.9.0)
 
-PATH
-  remote: engines/commercial/remove_vendor_branding
-  specs:
-    remove_vendor_branding (0.1.0)
-
```

Comment:
I had these changes after building, but I'm not 100% sure if they are supposed to be like this?

Reply:
This is not normal. Please take it out. If you see this again after pulling the master into your branch and rebuilding, I suggest to post a question about it in Slack.